src/shared/containers/SubmissionPage.jsx

```diff
@@ -22,7 +27,18 @@ class SubmissionsPageContainer extends React.Component {
     this.handleSubmit = this.handleSubmit.bind(this);
   }
 
-  componentDidMount() { }
+  componentDidMount() {
+    const {
+      auth,
+      communitiesList,
+      getCommunitiesList,
+    } = this.props;
+
+    if (!communitiesList.loadingUuid
+    && (Date.now() - communitiesList.timestamp > 10 * MIN)) {
+      getCommunitiesList(auth);
+    }
```

Comment:
Here, we only need to load / update `communitiesList` if challenge belongs to any groups.

Reply:
@birdofpreyru implemented in https://github.com/topcoder-platform/community-app/pull/1286